tests/integration/test_httpx.py

```diff
@@ -314,3 +319,19 @@ def test_redirect_wo_allow_redirects(do_request, mockbin, yml):
         assert response.status_code == 308
 
         assert cassette.play_count == 1
+
+@pytest.mark.online
+@pytest.mark.parametrize("url", ["https://github.com/kevin1024/vcrpy/issues/" + str(i) for i in range(3, 6)])
+def test_simple_stream_fetching(yml, url):
+    client = httpx.Client()
+    with vcr.use_cassette(yml):
+        with client.stream("GET", url):
+            pass
+
+    with vcr.use_cassette(yml) as cassette:
+        output = io.BytesIO()
+        with client.stream("GET", url) as response:
+            for chunk in response.iter_bytes():
+                output.write(chunk)
+            assert str(response.request.url) == url
+        assert cassette.play_count == 1
```

Comment:
These two asserts do not seem core to the test. If they are, let's briefly document the idea, if not, let's drop the two. If you like the idea of using `f'{httpbin.url}/bytes/<some number>'` (above) we could `assert len(output.getvalue()) == <some number>` here or compared against a pre-recorded body and add a self-test that the original matches that length, instead.

Reply:
You are right, the byte count is really the core test. I'll update that.